app/routes/app/brainstorm/$marketplace.challenges.tsx

```diff
@@ -0,0 +1,324 @@
+              <Text weight={500} color="dimmed" size="xs" className="mt-3">
+                HOW THE REWARD POOL IS DISTRIBUTED
+              </Text>
+              <div className="flex flex-row space-x-3 mt-1">
+                <Badge size="sm" radius="sm">
+                  Aggresive
+                </Badge>
+                <Text size="sm">Rewards the top 10% of submissions. Winners are determined through peer review</Text>
+              </div>
+            </Paper>
+          </div>
+        </div>
+      </main>
+    </section>
+  );
+}
+
+// Responsive layout for displaying marketplaces. On desktop, takes on a pseudo-table layout. On mobile, hide the header and become a list of self-contained cards.
+function MarketplacesChallengesTable({ challenges }: { challenges: ChallengeWithMarketplace[] }) {
+  if (challenges.length === 0) {
+    return <Text>No results. Try changing search and filter options.</Text>;
+  }
+  return (
+    <>
+      {/* Header (hide on mobile) */}
+      <div className="hidden lg:grid grid-cols-6 gap-x-1 items-end px-2">
+        <div className="col-span-2">
+          <Text color="dark.3">Challenge Marketplace</Text>
+        </div>
+        <Text color="dark.3">Chain/Project</Text>
+        <Text color="dark.3">Reward Pool</Text>
+        <Text color="dark.3">Submit Deadline</Text>
+        <Text color="dark.3">Review Deadline</Text>
+      </div>
+      {/* Rows */}
+      <div className="space-y-3">
+        {challenges.map((c) => {
+          return (
+            <Link
+              to="/app/brainstorm/c/[id]"
+              // On mobile, two column grid with "labels". On desktop hide the "labels".
+              className="grid grid-cols-2 lg:grid-cols-6 gap-y-3 gap-x-1 items-center border-solid border-2 border-[#EDEDED] px-2 py-5 rounded-lg hover:border-brand-400 hover:shadow-md shadow-sm"
+              key={c.id}
+            >
+              <div className="lg:hidden">Challenge Marketplaces</div>
+              <div className="lg:col-span-2">
+                <Text>{c.title}</Text>
+              </div>
+              <div className="lg:hidden">Chain/Project</div>
+              <ProjectWithIcon project={c.marketplace.project} />
+              <div className="lg:hidden">Reward Pool Totals</div>
+              <TextWithIcon text={`${c.marketplace.rewardPool.toLocaleString()} USD`} iconUrl="/img/icons/dollar.svg" />
+              <div className="lg:hidden">Submit Deadline</div>
+              <Text color="dark.3">{`${c.marketplace.endsAt?.toLocaleString()}`} </Text>
+              <div className="lg:hidden">Review Deadline</div>
+              <Text color="dark.3">{c.marketplace.reviewDeadline?.toLocaleString()}</Text>
+            </Link>
+          );
+        })}
+      </div>
+    </>
+  );
+}
+
+function ProjectWithIcon({ project }: { project: string }) {
+  const iconUrl = PROJECT_ICONS[project];
+
+  return <TextWithIcon text={project} iconUrl={iconUrl ?? null} />;
+}
```

Comment:
Maybe combine with/use Project badge since it's used elsewhere

Reply:
we could add this component to the ProjectBadge component. This I believe is a newer better looking version that Sebastian created for the Brainstorm page

Reply:
Oh I didn't realize it was in the table and the top badge. I think the top of page one probably want to be ProjectBadge unless we are changing the design then yeah combine